src/zenml/orchestrators/input_utils.py

```diff
@@ -115,8 +124,8 @@ def resolve_step_inputs(
         if issue_found:
             raise ValueError(
                 "Cannot fetch requested information from model "
-                f"`{config_.model.name}` version "
-                f"`{config_.model.version}` given artifact "
+                f"`{config_.model_name}` version "
+                f"`{config_.model_version}` given artifact "
```

Comment:
This might be `None` I guess, which we could probably display in a better way in the logs?

Reply:
Significantly reworked this part